bindings/pydeck/pydeck/io/html.py

```diff
@@ -87,14 +88,30 @@ def display_html(filename):
 
 
 def iframe_with_srcdoc(html_str, width="100%", height=500):
-    width = '"{}"'.format(width) if type(width) == str else width
-    iframe = """<iframe src="about:blank" frameborder="0" srcdoc="{}" width={} height={}></iframe>""".format(
-        html.escape(html_str), width, height
+    if isinstance(width, str):
+        width = '"{}"'.format(width)
```

Comment:
```suggestion
        width = f'"{width}"'
```
I think we require Python 3.6+ now, so we should default to using f-strings.

Reply:
Agreed!